en/docs/guides/user-mgt/managing-user-attributes.md

```diff
@@ -0,0 +1,204 @@
+        LDAP, using both **mail** and **uid** attributes.
+
+        ``` toml
+        [user_store]
+        user_name_search_filter = "(&amp;(objectClass=person)(uid=?))"
+        ```
+
+    2.  Disable the `user_dn_pattern` property if
+        it is currently enabled.
+
+        ``` toml
+        [user_store]
+        user_dn_pattern = "uid={0},ou=Users,dc=wso2,dc=org"
+        ```
+
+    3.  The `mail` attribute has unique requirements. If you are using the
+        `mail` attribute, add the following. For more information on email
+        authentication, see [Using Email Address as the
+        Username](../../guides/user-mgt/email-as-username)
+        .
+
+        ``` toml
+        [tenant_mgt]
+        enable_email_domain = true
+        ```
+
+2.  If you want to work with multiple attributes (basically to retrieve
+    internal roles with multiple attributes), do one of the following
+    depending on which user store you want to configure this for:
+
+    -   **Configuration for primary user store:** Add the following
+        property in the `<IS_HOME>/repository/conf/deployment.toml`
+
+        ``` toml
+        [user_store]
+        properties.MultipleAttributeEnable = true
+        ```
+
+3.  To test this, restart WSO2 IS and attempt to log in to the
+    user portal by providing either the `          mail         `
+    or `          uid         ` values with the same password.
+
+### Customizing the claim for the user attribute
+
+If you are using multiple attribute authentication and want to customize
+the claim to be used for user name attribute, do the following.
+
+Add the following configuration in the
+`         <IS_HOME>/repository/conf/deployment.toml        `
+file.
+
+``` toml
+[authentication.authenticator.basic] 
+name ="BasicAuthenticator"
+enable=true
+UserNameAttributeClaimUri = "http://wso2.org/claims/emailaddress"
+```
+
+This will return the email address of the authenticated user. It can be
+configured to return any attribute by changing the '
+`         UserNameAttributeClaimUri        ` ' parameter.
+
+!!! info "Related Topics"
+    For more information, see the following links.
+
+    -   For working with claim dialects and mapping claims, see [Claim
+        Management](insert-claim-management).
+    -   See [Configuring Claims for a Service
+        Provider](insert-configuring-claims-for-a-service-provider) on how to
+        configure claims when [adding a service
+        provider](insert-adding-and-configuring-a-service-provider).
+    -   See [Configuring Claims for an Identity
+        Provider](insert-configuring-claims-for-an-identity-provider) on how to
+        configure claims when [adding an identity
+        provider](insert-adding-and-configuring-an-identity-provider).
+    -   See [Writing a Custom User Store Manager](insert-writing-a-custom-user-store-manager) for instructions on
+        how to write a custom user store manager.
```

Comment:
the plan for claim management was to have fragments with steps related to things like "adding claims" "adding claim dialects" etc  and include them in the relevant pages...so these links may not be relevant with the new structure.

Reply:
check other related links as well please

Reply:
I have included many (insert-link) portions in the user-mgmt section in the other pages as well. Will update everything with the proper links or fragments, once we have finalized if it is going to go to a section or remain a link.

Reply:
+1